app/templates/view_org_data.html

```diff
@@ -35,7 +35,7 @@ <h2>Organization Config Table</h2>
             </li>
             <li class="usa-button-group__item">
                 <button class="usa-button usa-button--secondary"
-                    onclick="confirmDelete('/organization/delete/{{data.organization.id}}')">Delete</button>
+                    onclick="confirmDelete('/organization/config/delete/{{data.organization.id}}')">Delete</button>
```

Comment:
why /something/**config**/action/id, not /something/action/id?

Reply:
not very sure, it was defined in route earlier, we may double check with Tyler @btylerburton